### PR TITLE
Create AzurePurview.json to enable Azure Purview to be a data connector within Azure Sentinel.

### PR DESCRIPTION
**Pull request by libarson**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: DataConnectors/Purview/AzurePurview.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All scan logs",
+            "query": "PurviewScanStatusLogs\n | sort by TimeGenerated"
+        },
+        {
+            "description" : "View files that contain a specific classification (example shows Social Security Number)",
+            "query": "PurviewDataSensitivityLogs\n | where Classification contains \"Social Security Number\""
```
Comment (v-rucdu) [inline DataConnectors/Purview/AzurePurview.json:25]: Sample query is giving error while executing for **PurviewDataSensitivityLogs** as
**Failed to resolve table or column expression named 'PurviewDataSensitivityLogs'**

**Review by v-rucdu — CHANGES_REQUESTED** [review]

File: DataConnectors/Purview/AzurePurview.json (hunk 1 above)
Comment (libarson) [reply DataConnectors/Purview/AzurePurview.json:25]: The PurviewDataSensitivityLogs are still being onboarded, will update once the logs are ready and the sample query can be tested.

Comment (v-rucdu) [reply DataConnectors/Purview/AzurePurview.json:25]: Thanks for the Update Lisa.

Do you want us to wait for this PR to be merged until **PurviewDataSensitivityLogs** is onboarded?

Comment (libarson) [reply DataConnectors/Purview/AzurePurview.json:25]: Yes, we can wait until the logs are onboarded. Hopefully, it will happen soon.

Comment (libarson) [reply DataConnectors/Purview/AzurePurview.json:16]: We got the PurviewDataSensitivityLogs onboarded in canary and I was able to successfully test the sample queries. I also removed the PurviewScanStatusLogs as we will only have the PurviewDataSensitivityLogs for the private preview. We are ready to move forward with merging the PR.

Comment (v-jayakal) [reply DataConnectors/Purview/AzurePurview.json:16]: @libarson , Can you please tell us how to test these queries and in which LA workspace is this PurvviewDataSensitivityLogs onboarded? Still do not see that.
image

**Review by v-jayakal — CHANGES_REQUESTED** [review]

File: DataConnectors/Purview/AzurePurview.json (hunk 1 above)
Comment (libarson) [reply DataConnectors/Purview/AzurePurview.json:16]: @Saasri Can you help out in giving guidance on where the PurviewDataSensitivityLogs are onboarded and how to test them?

Comment (saasri) [reply DataConnectors/Purview/AzurePurview.json:16]: image

File: DataConnectors/Purview/AzurePurview.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        {
+            "metricName": "Total data received",
+            "legend": "PurviewDataSensitivityLogs",
+            "baseQuery": "PurviewDataSensitivityLogs"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "View files that contain a specific classification (example shows Social Security Number)",
+            "query": "PurviewDataSensitivityLogs\n | where Classification contains \"Social Security Number\""
```
Comment (saasri) [inline DataConnectors/Purview/AzurePurview.json:16]: Change 'contains' to 'has'

Comment (libarson) [reply DataConnectors/Purview/AzurePurview.json]: Updated

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**